check_pgactivity

```diff
@@ -542,6 +542,49 @@ sub find_hostkey($) {
     return "binary defaults";
 }
 
+# Record on disk some given data to a given file.
+#
+# To avoid data loss, the sub:
+# * first require an exclusive lock on the lock file,
+# * load the existing data
+# * call the closure to record its data in there
+# * write back the edited data to disk
+# * close the file and release the lock.
+#
+# The closure is reponsible to define how and where in the structure it wants to
+# record its data.
+#
+# Parameters are :
+#  * $rec_to: a closure responsible to record its data in the given structure ref
+#  * $storage: file to write to
+sub lock_save($$) {
+    my $rec_to   = shift;
```

Comment:
`$rec_to` isn't really self explanatory.  Maybe $fn_record or something like that?

Reply:
done